<?xml version="1.0"?>
<launch>
  <arg name="model_name" default="box" />
  <arg name="gripper_type" default="tercero_short" />
  <arg name="tf_name" default="grasp_point" />
  <rosparam file="$(find denso_gazebo_grasp)/config/grasp_offset.yaml" command="load" />

  <node pkg="denso_gazebo_grasp" type="grasp_point_tf_publisher" name="grasp_point_tf_publisher">
    <param name="model_name" value="$(arg model_name)"/>
    <param name="gripper_type" value="$(arg gripper_type)"/>
    <param name="tf_name" value="$(arg tf_name)" />
  </node>
</launch>